packages/arb-token-bridge-ui/src/components/Sidebar/SidebarStore.ts

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from 'react'
+import { useLocalStorage, useWindowSize } from 'react-use'
```

Comment:
We could use `useLocalStorage` from `@uidotdev/usehooks`, it's more reactive.

Reply:
also not changing it from portal